packages/ember-glimmer/tests/integration/content-test.js

```diff
@@ -59,6 +59,102 @@ moduleFor('Static content tests', class extends RenderingTest {
 
 });
 
+moduleFor('Null object tests', class extends RenderingTest {
+  ['@test it should read from an Object.create(null)']() {
+    let nullObject = Object.create(null);
+    nullObject['foo'] = 'bar';
+
+    this.render('{{nullObject.foo}}', { nullObject });
+
+    this.assertText('bar');
+
+    this.runTask(() => this.rerender());
+    this.assertText('bar');
+
+    this.runTask(() => set(this.context, 'nullObject.foo', 'baz'));
+    this.assertText('baz');
+
+    this.runTask(() => set(this.context, 'nullObject.foo', 'bar'));
+    this.assertText('bar');
+
+    let nullObject2 = Object.create(null);
+    nullObject2['foo'] = 'baz';
+
+    this.runTask(() => set(this.context, 'nullObject', nullObject2));
+    this.assertText('baz');
+  }
+});
+
+moduleFor('Escaped content tests', class extends RenderingTest {
+
+  ['@test it should read an escaped number value']() {
+    this.render('{{aNumber}},{{nested.aNumber}},{{{aNumber}}}', {
+      aNumber: 1,
+      nested: { aNumber: 3 }
+    });
+
+    this.assertText('1,3,1');
+
+    this.runTask(() => this.rerender());
+    this.assertText('1,3,1');
+
+    this.runTask(() => set(this.context, 'aNumber', 2));
+    this.assertText('2,3,2');
+
+    this.runTask(() => set(this.context, 'aNumber', 1));
+    this.assertText('1,3,1');
+
+    this.runTask(() => set(this.context, 'nested', { aNumber: 4 }));
+    this.assertText('1,4,1');
+  }
```

Comment:
If we make the "escaped" position part of the matrix, it should be covered by the shared test cases